packages/react/src/button/Button.tsx

```diff
@@ -70,9 +73,27 @@ export const Button = forwardRef<HTMLButtonElement, ButtonProps>(
         </Text>
       );
 
-    const presetProps = presets[preset];
+    const presetProps = appearances[appearance];
     const finalColorScheme = colorScheme ?? presetProps.colorScheme;
     const finalVariant = variant ?? presetProps.variant;
+    const content = icon ? (
+      <Box className={clsx(styles.icon({ size: size }))}>{icon}</Box>
+    ) : (
+      <Box
+        asChild
+        className={clsx(
+          styles.componentWrapper({
+            size: size,
+          }),
```

Comment:
As we are not merging any class name, we can just use `className={ styles.componentWrapper({
            size,
          })}`